crates/revm/revm-inspectors/src/tracing/mod.rs

```diff
@@ -60,9 +62,14 @@ impl TracingInspector {
         }
     }
 
-    /// Consumes the Inspector and returns the recorded.
-    pub fn finalize(self) -> CallTraceArena {
-        self.traces
+    /// Consumes the Inspector and returns a [ParityTraceBuilder].
+    pub fn into_parity_builder(self) -> ParityTraceBuilder {
+        ParityTraceBuilder::new(self.traces.arena, self.config)
```

Comment:
are traces within the arena sorted in the order of occurrence?

Reply:
yes